src/garage/sampler/off_policy_vectorized_sampler.py

```diff
@@ -60,6 +64,7 @@ def shutdown_worker(self):
         self.vec_env.close()
 
     def obtain_samples(self, itr, batch_size):
+        # pylint: disable=arguments-differ
```

Comment:
why?

Reply:
This signature conflicts with the parent class. I will try fix it.